packages/docs/docs/guides/models.md

```diff
@@ -1,17 +1,13 @@
 # Models
 
-AI.JSX supports [OpenAI](https://openai.com/) and [Anthropic](https://www.anthropic.com/).
-
 :::tip Getting started?
-Most people should start with OpenAI's GPT-4 model.
-
-If you need a very long context window, use Anthropic's Claude 100k.
+AI.JSX supports models from [OpenAI](https://openai.com/) and [Anthropic](https://www.anthropic.com/). Most people should start with OpenAI's GPT-4 model. If you need a very long context window, use Anthropic's Claude 100k.
 
-([More detail](./brand-new.md#recommended-dev-workflow))
+See the [recommended dev workflow](./brand-new.md#recommended-dev-workflow) content for tips on model selection.
```

Comment:
```suggestion
See the [recommended dev workflow](./brand-new.md#recommended-dev-workflow) guide for tips on model selection.
```

Or we could remove the word "guide" / "content" altogether.

Reply:
yeah i'm gonna just go w/your suggestion. i didn't use 'guide' before since it's technically a section in the n00b guide but close enough :)